src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.domain.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private static List<Fruit> fruits = new ArrayList<>();
```

Comment:
The 'fruits' field should be final to ensure that the reference to the list is not changed elsewhere in the code.